doc/developing-packages.rst

```diff
@@ -259,6 +259,54 @@ your libraries and executable sections. For example:
 Note that the ``import`` **must** be the first thing in the stanza. For more
 information see the :ref:`common-stanzas` section.
 
+Sets of packages that are verified to build together
+----------------------------------------------------
+
+Since Cabal 3.8, you can import from a
+:ref:`remote project configuration file<conditionals and imports>`:.
+This frees you from having to determine the acceptable version bounds for your
+dependencies, as described in the previous section.
+
+One provider of package sets is Stackage_. The Stackage snapshots contain a set
+of packages from Hackage that have all been verified to build with a given
+version of GHC.
+
+For example, the snapshot lts-19.2 contains versioned packages which all compile on GHC 9.0.
+
+If you use the following ``cabal.project`` file, the packages of the snapshot
+would be constrained to specific versions. The Cabal solver is then liberated
+from determining which versions to use, since it has already been determined
+which versions to use, at least for the packages in the snapshot.
+
+::
+
+    packages: .
+    import https://www.stackage.org/lts-19.2/cabal.config
```

Comment:
Not sure if it would be too much info but you could leverage if's to import several lts on ghc version in the same cabal.project

Reply:
Let's defer this to a future PR. I think it is nice to have a simple example.